mk_verificator/tests/networking/mtu/test_mtu.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+
+import salt.client as client
+import texttable as tt
+
+expectations = {
```

Comment:
let's merge utils first. after that I suggest to separate logic and move this large json to the divided file